parcel.go

```diff
@@ -13,48 +13,94 @@ func NewParcelStore(db *sql.DB) ParcelStore {
 }
 
 func (s ParcelStore) Add(p Parcel) (int, error) {
-	// реализуйте добавление строки в таблицу parcel, используйте данные из переменной p
-
-	// верните идентификатор последней добавленной записи
-	return 0, nil
+	// добавление строки в таблицу
+	res, err := s.db.Exec("insert into parcel (client, status, address, created_at) values (:client,:status, :address, :created_at)",
+		sql.Named("client", p.Client),
+		sql.Named("status", p.Status),
+		sql.Named("address", p.Address),
+		sql.Named("created_at", p.CreatedAt))
+
+	if err != nil {
+		return 0, err
+	}
+
+	// идентификатор последней добавленной записи
+	id, err := res.LastInsertId()
+	if err != nil {
+		return 0, err
+	}
+	return int(id), nil
 }
 
 func (s ParcelStore) Get(number int) (Parcel, error) {
-	// реализуйте чтение строки по заданному number
-	// здесь из таблицы должна вернуться только одна строка
-
-	// заполните объект Parcel данными из таблицы
+	// чтение строки по заданному number
+	// возвращается только одна строка
+	// заполняем Parcel данными из таблицы
 	p := Parcel{}
-
+	row := s.db.QueryRow("select number, client, status, address, created_at from Parcel where number =:number", sql.Named("number", number))
+	err := row.Scan(&p.Number, &p.Client, &p.Status, &p.Address, &p.CreatedAt)
+	if err != nil {
+		return p, err
+	}
 	return p, nil
 }
 
 func (s ParcelStore) GetByClient(client int) ([]Parcel, error) {
-	// реализуйте чтение строк из таблицы parcel по заданному client
-	// здесь из таблицы может вернуться несколько строк
-
-	// заполните срез Parcel данными из таблицы
+	// чтение строки по заданному client
+	// возвращается несколько строк
+	// заполняем Parcel данными из таблицы
 	var res []Parcel
 
+	rows, err := s.db.Query(
+		"SELECT number, client, status, address, created_at FROM parcel WHERE client = :client",
+		sql.Named("client", client))
+
+	if err != nil {
+		return res, err
```

Comment:
В случае ошибки лучше возвращать `nil`
```suggestion
		return nil, err
```